### ISSUE TITLE
Is freetubeapp.io blocked by Russian censors or does FreeTube block Russian users?

### ISSUE THREAD

**Issue by acoolstraw** [FreeTubeApp/FreeTube#2290]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a bug report that matches the one I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.

### Describe the bug

It appears to be unavailable in Russia
![2022-06-04_18-20](https://user-images.githubusercontent.com/45886986/172015954-b789684f-e9a4-4fec-b080-1245d0ce05e1.png)

### Expected Behavior

should be available in Russia

### Issue Labels

accessibility issue, feature stopped working

### FreeTube Version

.

### Operating System Version

.

### Last Known Working FreeTube Version (If Any)

.

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by acoolstraw** [comment]
uh, it worked again? :shrug:

**Comment by PrestonN** [comment]
Just to clarify, we don't do any blocking ourselves. If at some point there were any blocking, it would be from something outside of the app like YouTube, GitHub, or the website hosting provider. In any of those situations were to occur, it'd be outside of our control.

**Comment by acoolstraw** [comment]
it appeared to be the person's connection or something - when they pinged the IP, it said it's Connection Refused. quite weird.
